Code/Framework/AzTest/AzTest/Platform/iOS/platform_ios_files.cmake

```diff
@@ -9,6 +9,7 @@
 set(FILES
     ../Common/UnixLike/AzTest/ColorizedOutput_UnixLike.cpp
     ../Common/Unimplemented/AzTest/ScopedAutoTempDirectory_Unimplemented.cpp
+    ../Common/Unimplemented/AzTest/Utils_UnixLike.cpp
```

Comment:
The directory name after "Common" should be "Unix" instead of "Unimplemented". :)

Reply:
lol, thanks